concepticondata/conceptlists/List-2023-1308/convert.py

```diff
@@ -39,6 +39,7 @@
     ])
 
 valid_edges = collections.defaultdict(list)
+selected_edges = collections.defaultdict(list)
```

Comment:
Where does this come from now? Like the `common_edges` in a previous version, this variable doesn't seem to be used anywhere.